src/Mono.Android/Test/Java.Interop/JnienvTest.cs

```diff
@@ -373,32 +374,27 @@ public void MoarThreadingTests ()
 			Assert.IsNull (ignore_t2, string.Format ("No exception should be thrown [t2]! Got: {0}", ignore_t2));
 		}
 
-		[DllImport ("__Internal", CallingConvention = CallingConvention.Cdecl)]
-		static extern IntPtr monodroid_typemap_java_to_managed (string java);
-
 		[Test]
 		public void JavaToManagedTypeMapping ()
 		{
-			var m = monodroid_typemap_java_to_managed ("android/content/res/Resources");
-			Assert.AreNotEqual (IntPtr.Zero, m);
-			m = monodroid_typemap_java_to_managed ("this/type/does/not/exist");
-			Assert.AreEqual (IntPtr.Zero, m);
+			Type m = Java.Interop.TypeManager.GetJavaToManagedType ("android/content/res/Resources");
```

Comment:
The one thing to keep in mind is that the subsequent assert only works if `Android.Content.Res.Resources` is preserved by the linker.  Apparently it is -- the test passes! -- but I'm not sure offhand *why* that type is preserved.  It might make more sense to test for a type that *can't possibly* be linked away, like... `Java.Lang.Object` or `Java.Lang.String`.

Reply:
That's for a different PR though :)